packages/edit-site/src/components/sidebar-navigation-screen-template-part/style.scss

```diff
@@ -0,0 +1,3 @@
+.edit-site-sidebar-navigation-screen-template-part-navigation-menu__title.components-heading {
+	margin-bottom: $grid-unit-10;
```

Comment:
@jasmussen it seems strange to have a different spacing for the heading component just on this one screen, do you think we should make this change to all heading components in browse mode? Maybe in a different PR?

Reply:
Yes, @jameskoster is still hashing out the metrics of the heading and subheading components, but they should all be the same. Good catch.

Reply:
Ok, I'll open a followup for this :)